docs/docs/consumer-development/onboarding.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 4
+title: Onboarding Checklist
+---
+# Consumer Onboarding Checklist
+
+The following checklists will aid in onboarding a new consumer chain to replicated security.
+
+Additionally, you can check the [testnet repo](https://github.com/cosmos/testnets/blob/master/replicated-security/CONSUMER_LAUNCH_GUIDE.md) for a comprehensive guide on preparing and launching consumer chains.
+
+## 1. Complete testing & integration
+
+- [ ] test integration with gaia
+- [ ] test your protocol with supported relayer versions (minimum hermes 1.3)
+- [ ] reach out to the ICS team if you are facing issues
+
+## 2. Create an Onboarding Repository
+
+To help validators and other node runners onboard onto your chain, please prepare a repository with information on how to run your chain.
+
+This should include (at minumum):
+
+- [ ] genesis.json witout CCV data (before the propsal passes)
+- [ ] genesis.json with CCV data (after spawn time passes)
+- [ ] information about relevant seed/peer nodes you are running
+- [ ] relayer information (compatible versions)
+- [ ] copy of your governance proposal (as JSON)
+- [ ] a script showing how to start your chain and connect to peers (optional)
+- [ ] take feedback from other developers, validators and community regarding your onboarding repo and make improvements where applicable
+
+Example of such a repository can be found [here](https://github.com/cosmos/ics-testnets/tree/main/game-of-chains-2022/sputnik).
```

Comment:
Is the link supposed to be working already? (It's not, but looks like a realistic link, which made me wonder)

Reply:
It's a real link, I just pointed to the wrong repo:
- [real one](https://github.com/hyphacoop/ics-testnets/tree/main/game-of-chains-2022/sputnik)

Reply:
Fixed!